src/open_inwoner/pdc/models/category.py

```diff
@@ -45,6 +62,7 @@ class Category(MP_Node):
     )
 
     node_order_by = ["slug"]
+    objects = PublishedQueryset.as_manager()
```

Comment:
Afaik you should use a subclass of `MP_NodeQuerySet` or `MP_NodeManager` as MP_Node model manager

Reply:
Thanks for this. This is indeed what is recommended in the documentation. I changed the class to MP_NodeQuerySet for the custom queryset.